aspnetcore/grpc/browser/sample/Handler.cs

```diff
@@ -0,0 +1,10 @@
+#region snippet_1
```

Comment:
Create a real handler file

Reply:
Noting what we decided to do with this one:  The app example available at the moment demonstrating how to properly set this up does not match close enough the snippet provided that I can create a real handler file I can trust.  For now, going with JamesNK's code snippit as-is.  However for the startup.cs example I created a compilable full file given that JamesNK's code snippet for that one close matched the guiding app example.